users/management/commands/create_users.py

```diff
@@ -0,0 +1,87 @@
+import logging
+from django.core.management.base import BaseCommand
+from django.contrib.auth import get_user_model
+from users.models import UserRole
+
+logger = logging.getLogger(__name__)
+User = get_user_model()
+
+
+class Command(BaseCommand):
+    help = "Create default users with predefined roles (idempotent)."
+
+    def handle(self, *args, **options):
+        roles = {role.role: role for role in UserRole.objects.all()}
+        required_roles = ["admin", "moderator", "user"]
+
+        for r in required_roles:
+            if r not in roles:
+                role_obj, _ = UserRole.objects.get_or_create(role=r)
+                roles[r] = role_obj
+                self.stdout.write(self.style.SUCCESS(f"Role '{r}' created."))
+            else:
+                self.stdout.write(self.style.WARNING(f"Role '{r}' already exists."))
+
+        users_data = [
+            {
+                "email": "admin@example.com",
+                "first_name": "Admin",
+                "last_name": "User",
+                "password": "admin1234",
+                "role": roles["admin"],
+                "is_active": True,
+                "is_staff": True,
+                "is_superuser": True,
+            },
+            {
+                "email": "mod@example.com",
+                "first_name": "Moderator",
+                "last_name": "User",
+                "password": "mod1234",
+                "role": roles["moderator"],
+                "is_active": True,
+                "is_staff": False,
+                "is_superuser": False,
+            },
+            {
+                "email": "user1@example.com",
+                "first_name": "User1",
+                "last_name": "Test",
+                "password": "user1234",
+                "role": roles["user"],
+                "is_active": True,
+                "is_staff": False,
+                "is_superuser": False,
+            },
+            {
+                "email": "user2@example.com",
+                "first_name": "User2",
+                "last_name": "Test",
+                "password": "user1234",
+                "role": roles["user"],
+                "is_active": True,
+                "is_staff": False,
+                "is_superuser": False,
+            },
+        ]
```

Comment:
Hardcoded passwords pose a security risk and should not be used in production. Replace hardcoded passwords with environment variables or securely generated values.